src/pushNotifications/TokenHandling.ts

```diff
@@ -56,7 +57,14 @@ const removeTokenFromServer = async (token: string) => {
 
     // 204 means that it was a success on the server
     // 404 means that the token was already not on the server and can be treated as a success
-    return response.status === 204 || response.status === 404;
+    const isSuccess = response.status === 204 || response.status === 404;
+
+    if (!isSuccess) {
+      Alert.alert(texts.errors.errorTitle, texts.errors.noData);
+      return false;
```

Comment:
this extra return is not needed. we will return  `isSuccess` anyways, which will be false in this case. i will remove it.